### DISCUSSION TITLE
Version 2.9.0 Windows Installer and Upgrade

### DISCUSSION THREAD

**Discussion by Sinfonia97** [codeproject/CodeProject.AI-Server#142] [Q&A] [answered]
I've been trying with no luck to find a Windows installer.  Can anyone point me in the right direction for that installer?

Secondly, is it possible to upgrade from 2.5.6 to 2.9.0 or whichever version is currently available with a Windows installer?

Thanks in advance.

**Comment by sym0nd0** [comment, marked as answer]
https://codeproject.github.io/
